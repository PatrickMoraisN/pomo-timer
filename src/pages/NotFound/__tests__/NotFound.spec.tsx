import { render } from '@testing-library/react'
import { NotFound } from '../NotFound'

describe('NotFound', () => {
  it('should render correctly', () => {
    const { container } = render(<NotFound />)
    expect(container).toMatchSnapshot()
  })
})
