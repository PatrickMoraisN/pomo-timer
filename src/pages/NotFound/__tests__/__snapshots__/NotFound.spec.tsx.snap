// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render correctly 1`] = `
<div>
  <h1>
    404: Page not found!
  </h1>
</div>
`;
